CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json

```diff
@@ -0,0 +1,19 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8",
+	    "commitHash": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",

```

Comment:
```suggestion
	    "commitHash": "8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
```

Reply:
Thanks for the correction, we modified as suggested.